numba/tests/test_pycc.py

```diff
@@ -353,13 +353,52 @@ def run_python(args):
             """
         run_python(["-c", code])
 
+    def check_setup_nested_py(self, setup_py_file):
+        # Compute PYTHONPATH to ensure the child processes see this Numba
+        import numba
+        numba_path = os.path.abspath(os.path.dirname(
```

Comment:
This get's the path of the Numba source code checkout, right? I.e. on my machine this gives `'/Users/vhaenel/git/numba'` -- so the directory where the top-level `setup.py`, lives.

Reply:
I only copied all existing test sources (and adjusted them for nested case).